data/specs.json

```diff
@@ -33,7 +36,8 @@
     "interconnect": { "full_name": "Interconnect Type" },
     "encoders_decoders": { "full_name": "Encoders and Decoders" },
     "cuda_compute_capability": { "full_name": "CUDA Compute Capability" },
-    "power_consumption": { "full_name": "Power Consumption", "unit": "W" }
+    "power_consumption": { "full_name": "Power Consumption", "unit": "W" },
+    "area": { "full_name": "Area", "unit": "mm2" }
```

Comment:
To comply with NVIDIA data sheets, I suggest to name this field "Die Size" (die_size).